pkg/vm/builtins/ecdsa.go

```diff
@@ -0,0 +1,160 @@
+package builtins
+
+import (
+	"fmt"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	starkcurve "github.com/consensys/gnark-crypto/ecc/stark-curve"
+	ecdsa "github.com/consensys/gnark-crypto/ecc/stark-curve/ecdsa"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+const ECDSAName = "ecdsa"
+const cellsPerECDSA = 2
+
+type ECDSA struct {
+	signatures map[uint64]ecdsa.Signature
```

Comment:
What hint would add signatures here?

Reply:
From compiler output it's this hint  `ecdsa_builtin.add_signature(ids.ecdsa_ptr.address_, (ids.signature_r, ids.signature_s))`